keyboards/yushakobo/primer29/info.json

```diff
@@ -0,0 +1,63 @@
+{
+    "manufacturer": "yushakobo",
+    "keyboard_name": "primer29",
+    "maintainer": "yushakobo",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
```

Comment:
```suggestion
    "matrix_pins": {
        "custom_lite": true,
```

Reply:
This change makes it impossible to build, so it is being returned.